lib/kafka_impl/util.ex

```diff
@@ -41,4 +48,6 @@ defmodule KafkaImpl.Util do
   def kafka_ex_worker("0.8.0"), do: KafkaEx.Server0P8P0
   def kafka_ex_worker("0.8.2"), do: KafkaEx.Server0P8P2
   def kafka_ex_worker("0.9.0"), do: KafkaEx.Server0P9P0
+  def kafka_ex_worker("kayrock"), do: KafkaEx.New.Client
```

Comment:
What is the significance of these new workers? I mean what will they be helping us with out of curiosity.

Reply:
they re different implementations based on different version of kafka server.
in config user can set 
```elixir
config :kafka_ex, kafka_version: <ONE_OF_THE_AVAILABLE_VERSIONS>
```
where `<ONE_OF_THE_AVAILABLE_VERSIONS>` is the on from the list "0.8.0", "0.8.2", "0.9.0", "kayrock" or anything else (then it will pick default to `Server0P10AndLater`)

Actually this piece is "borrowed" from the [kafka_ex/config](https://github.com/kafkaex/kafka_ex/blob/0d222574a1178fcebca0824c193b86530d4ee8fb/lib/kafka_ex/config.ex#L93-L97)

Reply:
> what will they be helping us with out of curiosity.

On of our services is starts using `kayrock` implementation.. so we need to support it to make it work=)